.travis.yml

```diff
@@ -1,18 +1,18 @@
 language: php
-php:
-  - 7.1
-  - 7.2
-  - nightly
+sudo: false
 matrix:
   include:
+    - php: 7.1
```

Comment:
Duplicate

Reply:
I added this back under the matrix based on the current travis builds. 
currently there are 3 tests per php version. 
based on composer highest, lowest, and default. This was to keep those tests. 

Is the intent to only test lowest and highest?